includes/widgets/common.php

```diff
@@ -258,15 +258,12 @@ private function register_layout_section() {
 					'{{WRAPPER}}' => 'width: {{SIZE}}{{UNIT}}; max-width: {{SIZE}}{{UNIT}}',
 				],
 				'separator' => 'after',
-				// TODO: BC settings:
-				'condition' => $is_container_active ? [] : [ '_element_width' => 'initial' ],
-				'device_args' => $is_container_active
-					? []
-					: $this->get_responsive_device_args( [
-						'condition' => [
-							'_element_width_{{DEVICE}}' => [ 'initial' ],
-						],
-					] ),
+				'condition' => [ '_element_width' => 'initial' ],
+				'device_args' => $this->get_responsive_device_args( [
+					'condition' => [
+						'_element_width_{{DEVICE}}' => [ 'initial' ],
+					],
+				] ),
```

Comment:
That's reverted to the original behavior:

https://github.com/elementor/elementor/blob/master/includes/widgets/common.php#L1274-L1281